nemo/core/config/pytorch_lightning.py

```diff
@@ -35,6 +35,7 @@ class TrainerConfig:
         https://pytorch-lightning.readthedocs.io/en/latest/trainer.html#
     """
 
+    _target_: str = 'pytorch_lightning.Trainer'
```

Comment:
Why is this necessary? Earlier we can pass the config to the constructor easily, now we need to use Instantiate and passing it in simply will raise an error in PTL trainer constructor.. 

This reduces visibility of arguments in constructor of Trainer (which may change while the config is not updated). Please revert.

Reply:
We can revert it.